doc/nrf/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_suit_customize_dfu.rst

```diff
@@ -337,7 +337,7 @@ The examples below demonstrate the use of these variables and methods.
 Set component definition and memory ranges
 ------------------------------------------
 
-In the :file:`app_envelope_yaml.jinja2` (found `here <https://github.com/nrfconnect/suit-generator/blob/main/ncs/app_envelope.yaml.jinja2>`__), the component definition and memory ranges are filled out by using the ``edtlib`` object like so:
+In the :file:`app_envelope_yaml.jinja2` (found in the **suit-generator** `app envelope template_`), the component definition and memory ranges are filled out by using the ``edtlib`` object like so:
```

Comment:
Link not working
```suggestion
In the :file:`app_envelope_yaml.jinja2` (found in the **suit-generator** `app envelope template`_), the component definition and memory ranges are filled out by using the ``edtlib`` object like so:
```

Reply:
Okay, thanks - confirmed with team that link will not work and needs be replaced with file path - will fix.